app/views/pd/IframeModal.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <ModalDynamicPromotion
```

Comment:
why are we re-using ModalDynamicPromotion? I thought it would be used for only promotion modals. Probably a good idea to use common modal or create a new general modal

Reply:
Haha, i didn't mean to rename it. We are now technically replacing common modal with this one. I'm not sure, it was the idea when we set it up. Added it to the eng-sync to discuss